Req. 0.0 - The project shall in a Requirements.md page

ChronoKeeperStudios opened the pull request.

ChronoKeeperStudios committed:
Created a Requirements File

ChronoKeeperStudios committed:
Added Requirements for the Turn game

ChronoKeeperStudios committed:
Update Requirements.md

erikbuck commented on Requirements.md:
```diff
@@ -0,0 +1,63 @@
+
+Req. 6.0 (Jonathan): The game shall include a gradual enemy encounter system
+
+Req. 6.0.1 (Jonathan): The enemy encounter system shall have users battle enemies that are of a similar level to the Player
```
Please define "similar" in a brief introductory paragraph.

erikbuck commented on Requirements.md:
```diff
@@ -0,0 +1,63 @@
+
+Req. 6.0.1 (Jonathan): The enemy encounter system shall have users battle enemies that are of a similar level to the Player
+
+Req. 6.1 (Jonathan): The game shall have certain enemies remain un-encounterable until the Player reaches a certain level
```
This req. may be redundant with Req. 6.0.1. Consider restating Req. 6.0.1 as "The enemy encounter system shall only spawn/attack/present enemies that are of similar level to the player."

erikbuck commented on Requirements.md:
```diff
@@ -0,0 +1,63 @@
+
+### [User Story E](Features.md "User Stories")
+
+Req. 5.0 (Hayden): The game shall include a "Metapod"-type enemy
```
Define "Matapod" in brief introduction.

erikbuck commented on Requirements.md:
```diff
@@ -0,0 +1,63 @@
+
+Req. 4.0.1 (Jonathan): The game shall implement an armor system that reduces damage dealt to the Player
+
+Req. 4.1 (Jonathan): The game shall implement an X-Attack System
```
Define X-Attack System in a brief introduction. Consider stating req. as "The X-Attack system shall..."

HRRichter96 committed:
Update Requirements.md

Added the requested changes. Wasn't sure how to correctly update the X-attack, so I'll let Jonathan change it if he'd like it to be worded differently.
